src/sql/workbench/parts/notebook/models/notebookModel.ts

```diff
@@ -920,6 +929,24 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		this._connectionUrisToDispose = [];
 	}
 
+	// Track time it takes to render all markdown cells
+	private trackMarkdownTelemetry(cellContent: nb.ICellContents, cellModel: ICellModel): void {
+		if (cellContent && cellContent.cell_type === CellTypes.Markdown) {
+			this._textCellsLoading++;
+		}
+		this._register(cellModel.onLoaded((e) => {
+			this._textCellsLoading--;
+			if (this._textCellsLoading <= 0) {
+				if (this._notebookOptions.editorLoadedTimestamp) {
+					if (this._notebookOptions.editorLoadedTimestamp && this._notebookOptions.editorLoadedTimestamp > 0) {
+						let markdownRenderingTime = Date.now() - this._notebookOptions.editorLoadedTimestamp;
+						this.telemetryService.publicLog(TelemetryKeys.NotebookMarkdownRendered, { markdownRenderingEllapsed: markdownRenderingTime });
```

Comment:
spelling - markdownRenderingElapsed (would also suggesting making it markdownRenderingElapsedMs)

Reply:
cool, changed.

Check out the use of "ellapsed" in both ours and vs code's codebases. you'd be... shocked